network/p2p/p2pbuilder/libp2pNodeBuilder.go

```diff
@@ -44,6 +44,13 @@ import (
 	"github.com/onflow/flow-go/network/p2p/utils"
 )
 
+type RoutingSystemActivation bool
+
+const (
+	RoutingSystemActivationEnabled  RoutingSystemActivation = true
+	RoutingSystemActivationDisabled RoutingSystemActivation = false
```

Comment:
Would it be clearer to rename these to something that denotes the DHT server will be started. I.e `DhtServerEnabled`

Reply:
https://github.com/onflow/flow-go/pull/4800/commits/6247470cbf04fbeabf9abc062ce695e598d80b2b